feature-group-definitions/document-picture-in-picture.yml

```
@@ -1,3 +1,4 @@
 name: Document picture-in-picture
+description: The document picture-in-picture API makes it possible to open an always-on-top window that can be populated with arbitrary HTML content.
```

style nit, brevity

```suggestion
description: The document picture-in-picture API makes an always-on-top window from arbitrary HTML content.
```

---

I like it. How about "creates" instead of "makes":

```suggestion
description: The document picture-in-picture API creates an always-on-top window from arbitrary HTML content.
```

---

That's fine with me.